packages/core/src/scss/settings/foundation/forms/_settings.scss

```diff
@@ -35,7 +35,7 @@ $form-label-bottom-margin:           ($form-spacing / 4);
 $floating-chevron-top:               $fontSize-root * 1.15;
 $floating-form-field-spacing:        8px;
 $floating-label-border-color:        color("greys", "light");
-$floating-label-bg-color:            color("greys", "lightest");
+$floating-label-bg-color:            color("whites", "bright");
```

Comment:
Can we remove the inset box shadow on form fields as well please. cc @bc-charlesho